web/public/themes/nex-dark.css

```
@@ -0,0 +1,71 @@
+/* ═══════════════════════════════════════════════════════════════
+   WUPHF Theme: Nex Dark
+   Dark variant of the Nex theme. Structural rules live in nex.css
+   (scoped with [data-theme^="nex"]) and are inherited here.
+   This file only overrides the semantic token layer.
+   ═══════════════════════════════════════════════════════════════ */
+
+@import url('./nex.css');
```

Stylelint (17.9.0)

[error] Expected "url('./nex.css')" to be "'./nex.css'" (import-notation)

(import-notation)